### DISCUSSION TITLE
中州韵插件逗号句号显示不正确问题

### DISCUSSION THREAD

**Discussion by rouvz** [fcitx5-android/fcitx5-android#617] [Q&A | 问答]
使用的软件信息如下：

### 输入法
应用名：小企鹅输入法5
版本：0.0.9-0-g8dc51356
包名：org.fcitx.fcitx5.android

### 插件
应用名：小企鹅输入法5(中州韵插件)
版本：0.0.9-0-g8dc51356
包名：org.fcitx.fcitx5.android.plugin.rime

如图，使用自带“双拼”方案，虚拟键盘下方逗号句号可以切换为中文样式，使用“中州韵”安装的“雾凇拼音”方案则不会改变。

![Screenshot_20241031-112847_纯纯写作](https://github.com/user-attachments/assets/42b2d3ca-36f6-4775-a93a-1844993c9b38)
![Screenshot_20241031-112850_纯纯写作](https://github.com/user-attachments/assets/0ecddcd5-4e63-4e08-a45f-1c458ec56477)
![Screenshot_20241031-121321_纯纯写作](https://github.com/user-attachments/assets/f9c6feae-6628-4864-ae65-789a45cddaa6)

**Comment by rocka** [comment]
首先 librime 没有提供查询英文标点到中文标点映射的功能；其次，这个映射是用户可配置的，所以也不能内置一份默认映射。除非 librime 增加相关的 API ，否则这个问题就没法解决。
